draft-ietf-rats-corim.md

```diff
@@ -1101,24 +1101,92 @@ The `uint` and `text` types MUST NOT be interpreted in a global scope.
+The Reference Values Triple has the following structure:
 
 ~~~ cddl
 {::include cddl/reference-triple-record.cddl}
 ~~~
 
+The `reference-triple-record` has the following parameters:
+
+* `ref-env`: Search criteria that targets Evidence environments in the ACS.
+* `ref-claims`: Search criteria that targets Evidence Claims in the ACS.
+
+To process `reference-triple-record` both the `ref-env` and `ref-claims` are compared with entries from the ACS.
+If these conditions are met, the matching ACS entry is copied into a new ACS entry that is added to the ACS,
+but with the Reference Value Provider's authority.
+
 #### Endorsed Values Triple {#sec-comid-triple-endval}
 
-An Endorsed Values triple declares additional measurements to add to the ACS.
+The Endorsed Values Triple has the following structure:
 
 ~~~ cddl
 {::include cddl/endorsed-triple-record.cddl}
 ~~~
 
+The `endorsed-triple-record` has the following parameters:
+
+* `condition`: Search criteria applied to the ACS that dermines if the `endorsement` applies.
+* `endorsement`: Claims to be added to the ACS.
+
+To proess a `endorsed-triple-record` the `condition` is compared with entries from the ACS.
+If the condition is met, the `endorsement` is combined with the `environment-map` of the matched `condition` to form a new ACS entry.
+The new ACS entry is added to the ACS with the Endorser's authority.
+
+#### Conditional Endorsement Triple {#sec-comid-triple-cond-endors}
+
+The Conditional Endorsement Triple has the following structure:
+
+~~~ cddl
+{::include cddl/conditional-endorsement-triple-record.cddl}
+~~~
+
+~~~ cddl
+{::include cddl/stateful-environment-record.cddl}
+~~~
+
+The `conditional-endorsement-triple-record` has the following parameters:
+
+* `conditions`: Search criteria applied to the ACS that dermines if the `endorsements` apply.
+* `endorsements`: Sets of Claims to be added to the ACS.
+
+To process a `conditional-endorsement-triple-record` the `conditions` are compared with entries from the ACS.
+If all the conditions are met, the `endorsements` are added to the ACS with the Endorser's authority.
+
+#### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
+
+The Conditional Endorsement Series Triple has the following structure:
+
+~~~ cddl
+{::include cddl/conditional-endorsement-series-triple-record.cddl}
+~~~
+
+~~~ cddl
+{::include cddl/conditional-series-record.cddl}
+~~~
+
+The `conditional-endorsement-series-triple-record` has the following parameters:
+
+* `condition`: Search criteria applied to the ACS that dermines if the `series` applies.
+* `series`: A set of selection-addition tuples.
+
+The `conditional-series-record` has the following parameters:
+
+* `selection`: Search criteria applied to the ACS entries resulting from the matched `conditions`
+* `addition`: Claims to be added to the ACS is the `selection` is satisfied.
+
+To process a`conditional-endorsement-series-record` the `conditions` are compared with entries from the ACS.
```

Comment:
```suggestion
To process a `conditional-endorsement-series-record`, first the `condition` is compared with entries from the ACS.
```

Reply:
All of the conditions could be compared unless one of them matches.

Reply:
whitespace fix applied